test/integration/targets/tower_project/tasks/main.yml

```diff
@@ -1,10 +1,21 @@
-- name: Create an SCM Credential
+- name: Create a SCM Credential
   tower_credential:
     name: SCM Credential for Project
     organization: Default
     kind: scm
 
 - name: Create a Project
+  tower_project:
+    name: my-project
+    organization: Default
+    state: present
+  register: result
+
+- assert:
+    that:
+      - "result is changed"
+
+- name: Update a Project (to set the SCM Credential)
```

Comment:
nit pick, but it's confusing to use the word "Update", because updating a project typically means pulling newest code from source control

Reply:
I see, thanks. Would there be any other good words?

Reply:
I'd say just "modify"

Reply:
OK, I'll fix it now!